torch/csrc/distributed/rpc/types.h

```diff
@@ -51,6 +51,11 @@ struct TORCH_API SerializedPyObj final {
   const std::vector<at::Tensor> tensors_;
 };
 
+struct RpcAgentOptions {
```

Comment:
Question: why this is here instead of in `RpcAgent.h`?

Reply:
I think it's better to put it under `RpcAgent.h`. Moved.